src/lib.rs

```diff
@@ -49,21 +49,21 @@ use std::marker::PhantomData;
 use std::ops::{Deref, DerefMut};
 use std::path::{Path, PathBuf};
 use std::rc::Rc;
-use std::sync::mpsc::{Receiver, channel};
+use std::sync::mpsc::{channel, Receiver};
 use std::time::{Duration, Instant};
 
 /// Loadable object from either the file system or memory.
 ///
 /// An object can be loaded if it can output a `Loaded<_>`. It’s important to note that you’re not
 /// supposed to use that trait directly. Instead, you should use the `Store`’s functions.
-pub trait Load: 'static + Sized {
+pub trait Load<C>: 'static + Sized {
```

Comment:
Ok I’m reading through. So I guess I’ll find a `Store<C>` and `Storage<C>` later. I fail to see any other way to do this right now, so I’m okay with that change!